spec/factories/landmarks.rb

```diff
@@ -1,10 +1,14 @@
 FactoryBot.define do
   factory :landmark do
-    sequence(:name) { |n| "Landmark #{n}" }
-    sequence(:lat) { |n| 39.739 +  n * 0.001 }
-    sequence(:long) { |n|-104.9903 + n * 0.0015}
-    sequence(:address) { |n| "Address #{n}" }
+    name { Faker::Address.community }
+    lat { Faker::Address.latitude }
+    long { Faker::Address.longitude }
+    address { Faker::Address.full_address }
+    phone_number { Faker::PhoneNumber.phone_number }
     category { "park" }
-    sequence(:phone_number) { |n| "Phone #{n}" }
+    place_id { Faker::Crypto.sha1 }
+    website { Faker::Internet.url }
+    photo_reference { 'CmRaAAAA58Iy6Em8ucPC_-OCxKTJ-p7YKDBUQ4mgVZj4qlFN6j2JhXKSfteprF-Ckca73TjXaYUtNk-trEbhBCWoUcWWsO42KRap_rlw4teWz4hqBU57sAoTl3-myl_4xjOw3LmBEhBTUAINq_SeOPXlwudc8ccKGhSH-pCQxOO6rRE-O1zFKPW1-h3OqA' }
+    md5_hash { Faker::Crypto.md5 }
```

Comment:
In the next branch (when we use the md5_hash), we should make this factory generate the real hash of the data.

Reply:
Agree

Reply:
I've updated the factory to represent the real hash.